README.md

```diff
@@ -50,13 +50,17 @@ A spring bean:
 
    }
 ``` 
-##Configuration for Service Authentication filter
+## Configuration for Service Authentication filter
 The following values must be provided to enable a ServiceAuthFilter bean:
 ```yaml
 idam:
   s2s-authorised:
     services: microservice1, microservice2
 ```
+ServiceAuthFilter bean is OncePerRequestFilter filter that you can add to your filter chain to authorise service 
```

Comment:
shouldn't it be a 403? 401 means I can't authenticate you, 403 means I know who you are but you aren't allowed access?

Reply:
@smathangi and myself went back and forth on this, I think we kept it the way the previous filter was working. Now you again started discussion on semantics of this, I have changed the code accordingly to  http://tools.ietf.org/html/rfc7235#section-3.1 and http://tools.ietf.org/html/rfc7231#section-6.5.3.

Reply:
thanks for fixing it, looks better now

Reply:
```suggestion
ServiceAuthFilter bean is a `OncePerRequestFilter` filter that you can add to your filter chain to authorise service 
```